neqo-http3/src/connection_client.rs

```diff
@@ -3139,4 +3139,34 @@ mod tests {
         client.process(out.dgram(), now());
         assert_closed(&client, Error::HttpFrameUnexpected);
     }
+
+    #[test]
+    fn we_can_have_transort_stream_readable_event_after_we_have_read_all_data_from_a_stream() {
```

Comment:
I kinda like the long test function names that the other tests you added use, but this one is a little *too* verbose I think. Please omit "we_can_have", "we_have", "from_a_stream", and fix "transport" typo and this is good.